### PR TITLE
Unit test for java.time

### PR DESCRIPTION
**Pull request by gaurangkudale**
Added the unit test for java.time

### CODE REVIEW

**Comment by gaurangkudale** [comment]
Hi @cyrille-artho 
As you said I've done the changes and created the new pull request

**Comment by gaurangkudale** [comment]
@cyrille-artho Any Updates?

**Comment by gaurangkudale** [comment]
@cyrille-artho 
Any updates?

**Comment by gaurangkudale** [comment]
Hi @cyrille-artho,
Here is a quick update I've added some more easy and simple test cases where `java.time` package is used waiting for your review

File: src/tests/java8/java.timeTest/ClockTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+     private Function<String,ZoneId> withZoneId= (timeZoneId -> ZoneId.of(ZoneId.SHORT_IDS.get(timeZoneId)));
+
+    @Test
+    public void fixed_clock_test() throws Exception{
+        Instant fixedInstant  = Instant.parse("2014-03-06T21:27:31Z");
+        Clock fixedClock = Clock.fixed(fixedInstant, withZoneId.apply("EST"));
+
+        assertThat(fixedClock.instant(),is(fixedInstant));
+        assertThat(fixedClock.instant(), is(fixedInstant));
```
Comment (cyrille-artho) [inline src/tests/java8/java.timeTest/ClockTest.java:21]: No need to repeat the assertion here.

File: src/tests/java8/java.timeTest/ClockTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        long timeDifference = edt.until(est,ChronoUnit.HOURS);
+        assertThat(timeDifference,is(-1L));
+    }
+
+    @Test
+    public void tick_minute_clock_test() throws Exception {
+        Clock clock = Clock.tickMinutes(withZoneId.apply("EST"));
+        LocalTime lt = LocalTime.now(clock);
+        Thread.sleep(3000L);
```
Comment (cyrille-artho) [inline src/tests/java8/java.timeTest/ClockTest.java:53]: Remove the line with `sleep` here.

**Review by cyrille-artho — COMMENTED** [review]
Thank you for the new tests. They are very good overall, and I think they cover the functionality sufficiently. As long as no native code is invoked, the existing library functions can be used without problems.

There is one place where an assertion is repeated (lines 20, 21); please remove the duplicate. Other than that, the tests look good, and we can include them after that change.

**Comment by gaurangkudale** [comment]
Hi @cyrille-artho 
Changes are done and there is one more thing  I want to work on any javapathfinder project if you have any project available for me please let me know.
I'm really excited and looking forward to do more contributions to JPF and learning new things

**Comment by gaurangkudale** [comment]
Hi @cyrille-artho 
Any updates?

**Comment by gaurangkudale** [comment]
Please give me Any updates whether the changes are right or wrong?

**Comment by cyrille-artho** [comment]
As of new tasks, the most urgent tasks are to ensure that all key features in Java 11 work as they did in Java 8.
There are a couple of unit tests that fail on Java 11, and sometimes, a smaller test can help us narrow down the root cause and develop a fix.

**Comment by gaurangkudale** [comment]
> tract one timesta

Yeah that's Right I'LL remove that comment

**Comment by cyrille-artho** [comment]
One test fails now:
```
 java8.ClockTest > switch_clock_time_zone_test FAILED
    java.lang.AssertionError at ClockTest.java:32
```
You can check the CI build status to get this information, by following the "Details" link next to the red X that marks the failed build.

**Comment by gaurangkudale** [comment]
Hi@cyrille-artho 
 can you please review the code now?

**Comment by gaurangkudale** [comment]
> One test fails now:
> ```
>  java8.ClockTest > switch_clock_time_zone_test FAILED
>     java.lang.AssertionError at ClockTest.java:32
> ```
> You can check the CI build status to get this information, by following the "Details" link next to the red X that marks the failed build.

Actually I'm not familiar with CI 
I want to learn CI/CD from can you please guide me from where should I start

**Comment by cyrille-artho** [comment]
Great, it works now!
You can find a good short intro about CI on YouTube: https://www.youtube.com/watch?v=1er2cjUq1UI

**Pull request merged**